rfcs/055-prismic-api/README.md

```diff
@@ -0,0 +1,249 @@
+                ... // reuse the DigitalLocation model
+            }
+        }
+    ],
+    "contributors": [ // behind ?includes=contributors
+        {
+            ... // reuse the Contributor model
+        },
+        ...
+    ],
+    "image": {
+        "thumbnail": {
+            ... // reuse the DigitalLocation model
+        },
+        "caption": ""
+    },
+    "seasons": [
+        "",
+        ...
+    ],
+}
+```
+
+```http
+GET /exhibitions?query=foo&filters=bar&sort=baz
+{
+    "type": "ResultList",
+    "results": [
+        {
+            ... // as above
+        },
+        ...
+    ],
+    "pageSize": 100,
+    "totalResults": 1000,
+    "totalPages": 10,
+    "nextPage": "/exhibitions?query=foo&filters=bar&sort=baz&page=2",
+}
+```
+
+#### Filter
+
+- date
+- status
+- place
+- contributor
+- format (ie permanent, exhibition, installation)
+- season
+
+#### Sort
+
+- date
+- relevance
+
+### `/events`
+
+#### Example response
+
+```http
+GET /events/{id}
+{
+    "id": "",
+    "format": "",
+    "title": "",
+    "status": "",
+    "isOnline": false,
+    "availableOnline": false,
+    "audience": "",
+    "instantiations": [
+        {
+            "start": "", // ISO 8601 datetime
+            "end": "", // ISO 8601 datetime
+            "place": ""
+        }
+    ],
+    "contributors": [ // behind ?includes=contributors
+        {
+            ... // reuse the Contributor model
+        },
+        ...
+    ],
+    "image": {
+        "thumbnail": {
+            ... // reuse the DigitalLocation model
+        },
+        "caption": ""
+    },
+    "partOf": [
+        {
+            ... // reuse the partOf model for series and seasons
+        }
+    ]
+}
+```
+
+```http
+GET /events?query=foo&filters=bar&sort=baz
+{
+    "type": "ResultList",
+    "results": [
+        {
+            ... // as above
+        },
+        ...
+    ],
+    "pageSize": 100,
+    "totalResults": 1000,
+    "totalPages": 10,
+    "nextPage": "/events?query=foo&filters=bar&sort=baz&page=2",
+}
+```
+
+#### Filter
+
+- date
+- accessibility status (is it audio described, signed, etc)
+- location
+- format (eg discussion)
+- series
+- season
+- audience (eg schools)
+- contributor
+- isOnline?
+- availableOnline?
+
+#### Sort
+
+- date
+- relevance
+
+## Open questions
+
+- How do we represent functional content?
+- If we want to avoid maintaining multiple versions of the same code, should this work be done as an extension to the concepts API?
+- Works exist at [api.wellcomecollection.org/catalogue/v2/works](api.wellcomecollection.org/catalogue/v2/works), concepts at [/catalogue/v2/concepts](api.wellcomecollection.org/catalogue/v2/concepts), images at [/catalogue/v2/images](api.wellcomecollection.org/catalogue/v2/images).
+Stories etc aren't part of the catalogue - should the URL be different? Does `/content` work?
+- Should we include a `/series` endpoint as part of this work?
+
+## Next steps
+
+- Answer most of those questions ☝️
```

Comment:
> > @rcantin-w @harrisonpim We've come up against some issues recently, outside of search (and may be relevant to search too), which this API could potentially resolve. Perhaps we could discuss tomorrow? cc @davidpmccormick
> 
> It would be good to understand what these are and whether it makes sense to do now. Whilst I don't see us using this API to render content pages, I do think that we may end up using it for section pages as well as for search. We need to prioritise its use for search, but if there are cases that we can knock off at the same time, it would be good to know about them now so that we can make sure we incorporate.

I never know how to deal with comments that aren't threadable, so putting it here as I know we can have a thread. It was about two things:

1) https://github.com/wellcomecollection/docs/pull/90#discussion_r1115559913
2) An issue with the sorting order of Stories, where older stories coming from the old platform would never have the correct publishing date as they were published in Prismic later on. Therefore there's a field for a custom date that's used in the UI (`published_date`(?)) but I believe cannot be used for Sorting. They would like to have that considered as part of the API and maybe take advantage of it (@gestchild please correct me if I'm wrong).

Reply:
2 is interesting and the sort of problem we could eventually solve with this. I would think that we should use the content pipeline to identify these articles and then correct the date to keep things like filtering consistent.

Reply:
So one for @jamieparkinson's ETL side of considerations, and resolved on here.